README.md

```diff
@@ -377,40 +377,53 @@ interface CacheProfile
 
 ### Flexible caching with stale-while-revalidate (SWR)
 
-The package supports flexible caching with stale-while-revalidate behavior using explicit middleware. With SWR, cached responses have two time periods:
+For certain types of data, it can be useful to allow partially stale data to be served while the cached value is recalculated in the background, preventing some users from experiencing slow response times while cached values are calculated
+this is the stale while revalidate behaviour.
+You can find more info how Laravel uses SWR: https://laravel.com/docs/12.x/cache#swr
 
-- **Fresh period**: The cache is considered fresh and served immediately without revalidation
-- **Stale period**: The cache is served immediately but triggers a background refresh
 
-This means users always get instant responses, even when cache needs updating.
 
 #### Using flexible caching
 
-Configure flexible caching per route using the `flexible()` method:
+Configure flexible caching per route using the `flexible()` method of FlexibleCacheResponse middleware:
 
 ```php
-use Spatie\ResponseCache\Middlewares\CacheResponse;
-
-// Fresh for 3 minutes, then stale for 15 minutes
+use Spatie\ResponseCache\Middlewares\FlexibleCacheResponse;
+
+/* Simple example how to use the flexible method: 
+/* between 0-180 seconds we will always serve fresh data from the cache 
+/* after 180 seconds we will serve "old" data and send new data to recalculate to the server
```

Comment:
Here also make it more readable by adding a comma. "After 180 seconds, we will..." Also do this for any other comments.